framework/platform/bpl/include/bpl/bpl_err.h

```diff
@@ -27,69 +27,71 @@ extern "C" {
-        ERROR_CODE(BPL_ERR_SLAVE_WIFI_CREDENTIALS_SET_FAILED) /* 36 */ ERROR_CODE(                 \
-            BPL_ERR_SLAVE_POST_INIT_CONFIG_FAILED) /* 37 */                                        \
-        ERROR_CODE(BPL_ERR_SLAVE_TX_CHANGE_STATE_FAILED) /* 38 */ ERROR_CODE(                      \
-            BPL_ERR_SLAVE_UPDATE_CREDENTIALS_FAILED) /* 39 */                                      \
-        ERROR_CODE(BPL_ERR_SLAVE_FAILED_CONNECT_TO_PLATFORM_MANAGER) /* 40 */ ERROR_CODE(          \
-            BPL_ERR_SLAVE_PLATFORM_MANAGER_REGISTER_TIMEOUT) /* 41 */                              \
-        ERROR_CODE(BPL_ERR_SLAVE_SLAVE_BACKHAUL_MANAGER_DISCONNECTED) /* 42 */ ERROR_CODE(         \
-            BPL_ERR_SLAVE_STOPPED) /* 43 */ ERROR_CODE(BPL_ERR_AP_MANAGER_START) /* 44 */          \
-        ERROR_CODE(BPL_ERR_AP_MANAGER_DISCONNECTED)                              /* 45 */          \
-        ERROR_CODE(BPL_ERR_AP_MANAGER_HOSTAP_DISABLED) /* 46 */ ERROR_CODE(                        \
-            BPL_ERR_AP_MANAGER_ATTACH_FAIL) /* 47 */                                               \
-        ERROR_CODE(BPL_ERR_AP_MANAGER_SUDDEN_DETACH) /* 48 */ ERROR_CODE(                          \
-            BPL_ERR_AP_MANAGER_HAL_DISCONNECTED) /* 49 */                                          \
-        ERROR_CODE(BPL_ERR_AP_MANAGER_CAC_TIMEOUT) /* 50 */ ERROR_CODE(                            \
-            BPL_ERR_MONITOR_DISCONNECTED) /* 51 */                                                 \
-        ERROR_CODE(BPL_ERR_MONITOR_HOSTAP_DISABLED) /* 52 */ ERROR_CODE(                           \
-            BPL_ERR_MONITOR_ATTACH_FAIL) /* 53 */                                                  \
-        ERROR_CODE(BPL_ERR_MONITOR_SUDDEN_DETACH) /* 54 */ ERROR_CODE(                             \
-            BPL_ERR_MONITOR_HAL_DISCONNECTED) /* 55 */                                             \
-        ERROR_CODE(BPL_ERR_MONITOR_REPORT_PROCESS_FAIL) /* 56 */ ERROR_CODE(                       \
-            BPL_ERR_CONFIG_PLATFORM_REPORTED_INVALID_CONFIGURATION) /* 57 */                       \
-        ERROR_CODE(BPL_ERR_CONFIG_BACKHAUL_WIRED_INTERFACE_IS_UNSUPPORTED) /* 58 */ ERROR_CODE(    \
-            BPL_ERR_CONFIG_BACKHAUL_WIRELESS_INTERFACE_IS_UNSUPPORTED) /* 59 */                    \
-        ERROR_CODE(BPL_ERR_CONFIG_NO_VALID_BACKHAUL_INTERFACE) /* 60 */ ERROR_CODE(                \
-            BPL_ERR_CONFIG_BRIDGE_INTERFACE_IS_EMPTY) /* 61 */                                     \
-        ERROR_CODE(BPL_ERR_WIFI_CONFIGURATION_CHANGE_TIMEOUT) /* 62 */ ERROR_CODE(                 \
-            BPL_ERR_WATCHDOG_PROCESS_STUCK) /* 63 */ ERROR_CODE(BPL_ERR_WATCHDOG_PROCESS_ZOMBIE)   \
-                                                                                                   \
-        /* 74 */ ERROR_CODE(BPL_ERR_LAST)
+    ERROR_CODE(BPL_ERR_NONE)                                                                       \
```

Comment:
It took me a bit of figuring out what was going on here... Basically, the comment was removed, and this allowed clang-format to properly format it. An alternative would be to put the number as a second argument to the macro:

```
#define GENERATE_ERROR_ENUM(ENUM, VALUE) ENUM = VALUE,
#define GENERATE_ERROR_STRING(STRING, VALUE) #STRING "=" #VALUE,

#define FOREACH_ERROR_CODE(ERROR_CODE) \
    ERROR_CODE(BPL_ERR_NONE, 0) \
...
```

Anyway, this thing about the comment that is removed should have been part of the commit message.

(nitpick)

Reply:
Updated commit message accordingly. Didn't think it is worth the extra effort to manually add the enum value for each error code line, if we really need it we can do it in the future.